MiniSim/Views/Preferences.swift

```diff
@@ -11,8 +11,30 @@ import Preferences
 import SwiftUI
 
 struct Preferences: View {
+    @State private var preferedTerminal: Terminal
+
+    init() {
+           let userPrefered = UserDefaults.standard.preferedTerminal
+           if let terminal = Terminal(rawValue: userPrefered ?? Terminal.terminal.rawValue) {
+               preferedTerminal = terminal
+           } else {
+               preferedTerminal = Terminal.terminal
+           }
+    }
+
     var body: some View {
         Settings.Container(contentWidth: 400) {
+            Settings.Section(title: "Preferred Terminal:") {
+                Picker("", selection: $preferedTerminal) {
+                    ForEach(Terminal.allCases, id: \.self) { terminal in
```

Comment:
I think we need to check whether this app is installed on users device. Now he can pick an app that's not available

Reply:
Yes, We have to check, I missed that one. Will work on that

Reply:
Yes we need to check. I missed that one. Will work on that